integration/zero-schema.gen.ts

```diff
@@ -8916,451 +8916,629 @@ export type Schema = typeof schema;
  * This type is auto-generated from your Drizzle schema definition.
+ *
+ * @deprecated Use Row["testSerialPk"] instead from "@rocicorp/zero".
  */
-export type TestSerialPk = Row<typeof testSerialPkTable>;
+export type TestSerialPk = Row["testSerialPk"];
 /**
  * Represents a row from the "testTextDefaultPk" table.
  * This type is auto-generated from your Drizzle schema definition.
+ *
+ * @deprecated Use Row["testTextDefaultPk"] instead from "@rocicorp/zero".
  */
-export type TestTextDefaultPk = Row<typeof testTextDefaultPkTable>;
+export type TestTextDefaultPk = Row["testTextDefaultPk"];
 /**
  * Represents a row from the "testTimestampDefaultPk" table.
  * This type is auto-generated from your Drizzle schema definition.
+ *
+ * @deprecated Use Row["testTimestampDefaultPk"] instead from "@rocicorp/zero".
  */
-export type TestTimestampDefaultPk = Row<typeof testTimestampDefaultPkTable>;
+export type TestTimestampDefaultPk = Row["testTimestampDefaultPk"];
 /**
  * Represents a row from the "testUuidPk" table.
  * This type is auto-generated from your Drizzle schema definition.
+ *
+ * @deprecated Use Row["testUuidPk"] instead from "@rocicorp/zero".
  */
-export type TestUuidPk = Row<typeof testUuidPkTable>;
+export type TestUuidPk = Row["testUuidPk"];
 /**
  * Represents a row from the "testUuidSqlDefaultPk" table.
  * This type is auto-generated from your Drizzle schema definition.
+ *
+ * @deprecated Use Row["testUuidSqlDefaultPk"] instead from "@rocicorp/zero".
  */
-export type TestUuidSqlDefaultPk = Row<typeof testUuidSqlDefaultPkTable>;
+export type TestUuidSqlDefaultPk = Row["testUuidSqlDefaultPk"];
 /**
  * Represents a row from the "timeEntry" table.
  * This type is auto-generated from your Drizzle schema definition.
+ *
+ * @deprecated Use Row["timeEntry"] instead from "@rocicorp/zero".
  */
-export type TimeEntry = Row<typeof timeEntryTable>;
+export type TimeEntry = Row["timeEntry"];
 /**
  * Represents a row from the "timesheet" table.
  * This type is auto-generated from your Drizzle schema definition.
+ *
+ * @deprecated Use Row["timesheet"] instead from "@rocicorp/zero".
  */
-export type Timesheet = Row<typeof timesheetTable>;
+export type Timesheet = Row["timesheet"];
 /**
  * Represents a row from the "user" table.
  * This type is auto-generated from your Drizzle schema definition.
+ *
+ * @deprecated Use Row["user"] instead from "@rocicorp/zero".
  */
-export type User = Row<typeof userTable>;
+export type User = Row["user"];
 
+/**
+ * Represents the ZQL query builder.
+ * This type is auto-generated from your Drizzle schema definition.
+ */
+export const zql = createBuilder(schema);
 /**
  * Represents the Zero schema query builder.
  * This type is auto-generated from your Drizzle schema definition.
+ *
+ * @deprecated Use `zql` instead.
  */
-export const builder = createBuilder(schema);
+export const builder = zql;
+
+/** Defines the default types for Zero */
+declare module "@rocicorp/zero" {
```

Comment:
We now automatically generate the `declare module` to define the schema, so one less step for users.